src/Symfony/Bundle/FrameworkBundle/Resources/config/http_client.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="http_client" class="Symfony\Contracts\HttpClient\HttpClientInterface">
+            <factory class="Symfony\Component\HttpClient\HttpClient" method="create" />
+            <argument type="collection" /> <!-- default options -->
+            <argument /> <!-- max host connections -->
+        </service>
+        <service id="Symfony\Contracts\HttpClient\HttpClientInterface" alias="http_client" />
+
+        <service id="psr18.http_client" class="Symfony\Component\HttpClient\Psr18Client" autowire="true">
```

Comment:
Do we really want to use autowiring here?

Reply:
see above